backend/index.php

```diff
@@ -3,15 +3,22 @@
 declare(strict_types=1);
 require __DIR__ . "/../vendor/autoload.php";
 
+$root = __DIR__ . "/../";
+
+use Dotenv\Dotenv;
 use Internships\Application\Application;
 
+$dotenv = Dotenv::createImmutable($root);
+$dotenv->load();
+
 $container = require __DIR__ . "/Application/AppContainerBuilder.php";
 $application = $container->get(Application::class);
-
-$options = getopt("r");
+$options = getopt("rf");
 
 if (array_key_exists("r", $options)) {
     $application->populate();
+} elseif (array_key_exists("f", $options)) {
+    $application->fetch();
```

Comment:
I prefer rename these options to some more readable names. Because these names `r` and `f` tell me nothing.
```
-r => -populate
-f => -fetch
```

Reply:
Oh, I wanted to do that, but guide I used didn't mention that I need to use two dashes as an option, so it didn't work before. I'm certainly making this change.